### PR TITLE
Use terminaltables for prettier output

### LINKED ISSUE #34: Suggestion: use terminaltables for prettier output
Currently the output is as follows:

```
--------- Caterpie ---------
CP 56 - 0/9/15 53.33%
CP 45 - 0/8/11 42.22%
CP 34 - 0/6/12 40.00%
CP 14 - 0/5/12 37.78%
CP 112 - 0/2/15 37.78%
CP 93 - 0/10/3 28.89%
CP 53 - 0/3/10 28.89%
CP 43 - 0/6/5 24.44%
CP 33 - 0/4/5 20.00%
CP 70 - 0/6/0 13.33%

--------- Weedle ---------
CP 49 - 1/14/12 60.00%
CP 70 - 1/14/10 55.56%
CP 10 - 1/12/0 28.89%
CP 46 - 9/0/2 24.44%
CP 24 - 1/0/9 22.22%
CP 33 - 2/2/3 15.56%

--------- Pidgey ---------
CP 10 - 5/14/12 68.89%
CP 62 - 4/10/14 62.22%
CP 60 - 2/11/13 57.78%
CP 77 - 2/12/11 55.56%
CP 10 - 2/8/9 42.22%
CP 41 - 2/2/15 42.22%
CP 105 - 0/9/5 31.11%
CP 88 - 0/9/5 31.11%
CP 10 - 2/9/3 31.11%
CP 24 - 6/1/6 28.89%
CP 22 - 0/1/4 11.11%
CP 82 - 0/0/1 2.22%
```

It looks like this data might be better presented in table form so that the “columns” line up nicely. Something like:

```
|---------|-------|-----|-----|-----|--------|
| Pokémon |   CP  | ATK | DEF | STA |  IV %  |
|---------|-------|-----|-----|-----|--------|
| Weedle  | CP 49 |  1  |  14 |  12 | 60.00% |
| Weedle  | CP 70 |  1  |  12 |   0 | 28.89% |
| Weedle  | CP 46 |  9  |   0 |   2 | 24.44% |
| Weedle  | CP 24 |  1  |   0 |   9 | 22.22% |
| Weedle  | CP 33 |  2  |   2 |   2 | 15.56% |
|---------|-------|-----|-----|-----|--------|
```

https://github.com/Robpol86/terminaltables can be used to create such ASCII tables, or even fancier ones.

### PR DESCRIPTION
**Pull request by tegob**
Closes #34.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -36,7 +37,6 @@ def init_config(self):
         parser.add_argument("-l", "--locale", default="en")
         parser.add_argument("--min_delay", type=int, default=10)
         parser.add_argument("--max_delay", type=int, default=20)
-        parser.add_argument("--iv", type=int, default=0)
```
Comment (tegob) [inline main.py:39]: Whoops; don’t merge!

**Comment by tegob** [comment]
Ok to merge now!

**Comment by tegob** [comment]
Example output using an [AsciiTable](https://robpol86.github.io/terminaltables/asciitable.html) (i.e. the current output with this patch):

```
+------------+-----+------+-----+-----+-----+
| Pokemon    |  CP | IV % | ATK | DEF | STA |
+------------+-----+------+-----+-----+-----+
| Bulbasaur  |  14 |  67% |  10 |  10 |  10 |
| Charmander |  36 |  56% |   0 |  14 |  11 |
| Squirtle   | 113 |  42% |   0 |  13 |   6 |
| Squirtle   | 135 |  20% |   0 |   1 |   8 |
| Caterpie   |  56 |  53% |   0 |   9 |  15 |
| Caterpie   |  45 |  42% |   0 |   8 |  11 |
+------------+-----+------+-----+-----+-----+
```

Example output using a [SingleTable](https://robpol86.github.io/terminaltables/singletable.html):

```
┌────────────┬─────┬──────┬─────┬─────┬─────┐
│ Pokemon    │  CP │ IV % │ ATK │ DEF │ STA │
├────────────┼─────┼──────┼─────┼─────┼─────┤
│ Bulbasaur  │  14 │  67% │  10 │  10 │  10 │
│ Charmander │  36 │  56% │   0 │  14 │  11 │
│ Squirtle   │ 113 │  42% │   0 │  13 │   6 │
│ Squirtle   │ 135 │  20% │   0 │   1 │   8 │
│ Caterpie   │  56 │  53% │   0 │   9 │  15 │
│ Caterpie   │  45 │  42% │   0 │   8 │  11 │
└────────────┴─────┴──────┴─────┴─────┴─────┘
```

File: main.py (hunk 1 above)
Comment (tegob) [inline main.py:39]: Fixed now.

**Comment by tegob** [comment]
@Boren Let me know if you want me to stop spamming you with PRs ;) I can start my own fork.

**Comment by Boren** [comment]
I'm just a bit busy right now, I'll merge when I get time

**Pull request merged**